frontend/src/css/terminal.scss

```diff
@@ -113,7 +113,7 @@
 }
 .terminal .inverted,
 .cmd .inverted,
```

Comment:
Not really sure of the context of .cmd but just following patterns, do you need a .cmd-inverted here?